src/main/java/seedu/letsgethired/logic/commands/NoteDeleteCommand.java

```diff
@@ -0,0 +1,70 @@
+package seedu.letsgethired.logic.commands;
+
+import static seedu.letsgethired.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.letsgethired.model.Model.PREDICATE_SHOW_ALL_APPLICATIONS;
+
+import java.util.List;
+
+import seedu.letsgethired.commons.core.index.Index;
+import seedu.letsgethired.logic.Messages;
+import seedu.letsgethired.logic.commands.exceptions.CommandException;
+import seedu.letsgethired.model.Model;
+import seedu.letsgethired.model.application.InternApplication;
+
+/**
+ * Deletes a note to an existing intern application in the interntracker.
+ */
+public class NoteDeleteCommand extends NoteCommand {
+    public static final String MESSAGE_ADD_NOTE_SUCCESS = "Deleted note from Intern Application";
+    private final Index index;
+    private final Integer targetIndex;
+
+    /**
+     * @param index of the intern application in the filtered intern application list to edit
+     * @param targetIndex  index number of the application in the list to delete
+     */
+    public NoteDeleteCommand(Index index, Integer targetIndex) {
```

Comment:
Minor nit: Perhaps we can make the naming a bit clearer here, with `internApplicationIndex` and `noteIndex`.
Also is the JavaDoc comment for `targetIndex` supposed to be "index number of **note** in the list to delete"?